tasks/agent.py

```diff
@@ -288,6 +289,9 @@ def omnibus_build(ctx, puppy=False, log_level="info", base_dir=None, gem_path=No
         overrides_cmd = "--override=" + " ".join(overrides)
 
     with ctx.cd("omnibus"):
+        # make sure bundle install starts from a clean state
+        ctx.run ("rm -f Gemfile.lock")
```

Comment:
not sure `rm` will work on windows (depends on environment). Perhaps the python file delete?